middlewares/user-serializer.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Or you can place them (this and future serializers) on the a new serializers folder.